archetypes/projects.md

```diff
@@ -0,0 +1,11 @@
++++
+title = "{{ replace .Name "-" " " | title }}"
+dates = ""
+description = ""
+cover_image = ""
+featured_image = ""
+summary = ""
+technologies = []
+website = ""
+git_repo = ""
```

Comment:
Some points about these front matter items:

- I think `dates` are not applicable to projects.
- What is the difference between `cover_image` and `featured_image`? `Cover` for cards and `feature` for project pages?
- `technologies` is a feature that looks like something very particular for your blog.

I think we can remove `technologies` and `dates`. The suit more as your own customization.

Reply:
I added dates and technologies because I use them for projects but I agree with you, it's maybe specific to my portfolio.

It's the opposite, `cover_image` is used at the top of a project page and `featured_image` is used for cards.